tests/test_component.py

```diff
@@ -206,7 +206,29 @@ class Media:
         )
 
     @parametrize_context_behavior(["django", "isolated"])
-    def test_template_name_static(self):
+    def test_template_file_static(self):
+        class SimpleComponent(Component):
+            template_file = "simple_template.html"
+
+            def get_context_data(self, variable=None):
+                return {
+                    "variable": variable,
+                }
+
+            class Media:
+                css = "style.css"
+                js = "script.js"
+
+        rendered = SimpleComponent.render(kwargs={"variable": "test"})
+        self.assertHTMLEqual(
+            rendered,
+            """
+            Variable: <strong data-djc-id-a1bc3e>test</strong>
+            """,
+        )
+
+    @parametrize_context_behavior(["django", "isolated"])
+    def test_template_file_static__compat(self):
```

Comment:
The diff is wrong, the added that tests the backwards compatibility test is this one below